src/components/home/components/styles.css

```diff
@@ -79,6 +79,18 @@
 /*
  this is for Navibar.js and home.js
   */
+
+  .avatar-dropdown {
+     background-color: #f56a00;
+     font-size: 80;
+     margin-left: 630px;
```

Comment:
this `margin-left: 630px` is breaking the dynamic flow of the app. Figure out a way to move this content to the right without a hard pixel shift